test/core/unreached-invalid.wast

```diff
@@ -623,3 +623,11 @@
   ))
   "type mismatch"
 )
+
+(assert_invalid
+  (module (func $tee-local-unreachable-value
+    (local i32)
+    tee_local 0 (unreachable))
```

Comment:
Actually, the bug in Wabt seems to be for `(unreachable) (tee_local 0)`, while this test is the other way round. Is that intentional?